web/components/space-homepage/SpaceLandingPage.tsx

```diff
@@ -110,16 +114,79 @@ export function SpaceLandingPage() {
     [profileListingData?.profile_listing]
   );
 
+  
   const filteredProfileListings = useMemo(() => {
     if (searchQuery === "")
       return shuffleProfiles(allProfileListings, getDayOfYear(new Date()));
 
     const searchQueryLower = searchQuery.toLowerCase();
     const fuse = new Fuse(allProfileListings, FUSE_OPTIONS);
-
     return fuse.search(searchQueryLower).map((result) => result.item);
   }, [allProfileListings, searchQuery]);
 
+ 
+  
+  
+
+  const { userData } = useUserData();
+  const [{ data: profileData }] = useAllProfilesOfUserQuery({
+    variables: { user_id: userData?.id ?? "" },
+  });
+
+
+
+  const idsToProfileScores = useMemo(() => {
+    let tempCounter = 0;
+    const tempIdsToProfileScores = new Map();
+    
+    // loop through all profiles in directory
+    for(let i = 0; i < filteredProfileListings.length; i++){
```

Comment:
Are you using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) to format your code?

[Guide](https://www.robinwieruch.de/how-to-use-prettier-vscode/) on setting it up